Hero.cpp

```diff
@@ -42,16 +42,17 @@ void Hero::attack(Monster& monster){
 
 Hero Hero::parse(const std::string& fileName){
     JSON json = JSON::parseFromFile(fileName);
-    
     return Hero(
         json.get<int>("base_health_points"),
-        json.get<int>("base_damage"),
+        json.get<int>("damage"),
+        json.get<int>("magical-damage"),
         json.get<int>("base_defense"),
         json.get<std::string>("name"),
         json.get<float>("base_attack_cooldown"),
         json.get<int>("experience_per_level"),
         json.get<int>("health_point_bonus_per_level"),
-        json.get<int>("damage_bonus_per_level"),
+        json.get<int>("physical_damage_bonus_per_level"),
+        json.get<int>("magic_damage_bonus_per_level"),
```

Comment:
Mindket damage, ha nem kap erteket, 0-nak kene lennie, meg szerintem a szintenkenti bonuszok is rossz erteket kapnak.

Reply:
Köszi az észrevételt, javítottam